### ISSUE TITLE
CasManager::isAuthenticated() only checks locally, not globally

### ISSUE THREAD

**Issue by defrox** [subfission/cas#51]
CasManager::isAuthenticated() only checks locally within laravel app if user has authenticated, but not globally with CAS.
If app A (laravel) hasn't authenticated and app B has already authenticated and CAS has a auth session, when CasManager::isAuthenticated() is called from app A, it will return FALSE but it should return TRUE as the user has already been authenticated through another app.
This will seamlessly login in users between different apps (not happening right now).

**Comment by subfission** [comment]
Great find!  checkAuthentication() is the intended method, although isAuthenticated() has similar effects.  For the moment, you can adjust your middleware to use the correct method.

I will patch the demo middleware accordingly and update the docs.

Update your middleware using the correct method like this:

```
if($this->cas->checkAuthentication())
{
	return redirect(config('cas.cas_redirect_path'));
}
```

**Comment by subfission** [comment]
Fixed with #52 fff20c4.  Feel free to check this over and see if it works as you  need.

Others that leverage the old method must not be impacted negatively.
